app/main.cpp

```diff
@@ -0,0 +1,47 @@
+#include <iostream>
+
+#include <bw64/bw64.hpp>
+
+#include "renderer.hpp"
+#include "parser.hpp"
+
+using namespace admrenderer;
+
+int dumpBw64AdmFile(const char* path) {
+  auto bw64File = bw64::readFile(path);
+  displayBw64FileInfos(bw64File);
+  displayAdmDocument(getAdmDocument(parseAdmXmlChunk(bw64File)));
+  displayChnaChunk(parseAdmChnaChunk(bw64File));
+  return 0;
+}
+
+int renderAdmContent(const char* input, const char* destination, const float dialogueGain = 1.0) {
+  auto bw64File = bw64::readFile(input);
+  const std::string outputDirectory(destination);
+  const std::string outputLayout("0+2+0"); // TODO: get it from args
+  Renderer renderer(bw64File, outputLayout, outputDirectory, dialogueGain);
+  renderer.process();
+  return 0;
+}
+
+void displayUsage(const char* application) {
+  std::cout << "Usage: " << application << " INPUT OUTPUT [GAIN]" << std::endl;
```

Comment:
So OUTPUT is optional as we can dump ?
`INPUT [OUTPUT] [GAIN]`

Reply:
Fixed in 8862894